indico/modules/rb_new/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -52,6 +52,7 @@ class BookingEditForm extends React.Component {
         hideOptions: {},
     };
 
+
```

Comment:
Is this intended? If so, why?

Reply:
I don't see a reason why that newline would be intended. :P Obviously a miss, thanks for pointing out!